zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -631,7 +631,7 @@ function ResultCtrl ($scope, $rootScope, $route, $window, $routeParams, $locatio
     let newParagraphConfig = angular.copy(paragraph.config)
     newParagraphConfig.results = newParagraphConfig.results || []
     newParagraphConfig.results[resultIndex] = config
-    if ($scope.revisionView === true) {
+    if ($scope.revisionView === true || $scope.viewOnly) {
```

Comment:
whoops, you're right :-) I missed that from our internal commit. Fixing this branch, will test locally and get back to you. Thanks for the patience!